### PR TITLE
Limits enforcement tweak

### PR DESCRIPTION
**Pull request by matteocargnelutti**
- Check capture state while steps are running so they can be interrupted on the fly if necessary.
- New `--attachments-bypass-limits` option: Consolidated behind an option flag a behavior allowing attachments to skip time and size constraints.
- Moved _"Capture Page Info"_ earlier in the capture process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ scoop "https://lil.law.harvard.edu" --capture-timeout 90000 --load-timeout 60000
   <summary><strong>See: Output of scoop --help 🔍</strong></summary>
 
 ```
-Usage: scoop [options] <url>
+Usage: @harvard-lil/scoop [options] <url>
```
Comment (bensteinberg) [inline README.md:157]: Is this right? Can you run on the command line with the `@harvard-lil/` prefix? It's not working in the VM I'm fussing with.

Comment (matteocargnelutti) [reply README.md:157]: It's not. Was automatically added from `package.json`, will fix! Good catch :D

**Comment by matteocargnelutti** [comment]
@bensteinberg Tests are no longer hanging 🥳 .

**Review by bensteinberg — APPROVED** [review]
🔥

**Pull request merged**